src/styles/create/index.scss

```diff
@@ -12,12 +12,17 @@
   color: var(--text-primary);
 }
 
-.wizard-create .ant-input,
-.ant-input-affix-wrapper,
-.ant-input-number,
-.ant-picker {
-  background-color: var(--background-l3);
-  border-color: var(--stroke-disabled);
+.wizard-create,
+.wizard-create-modal {
+  .ant-input,
+  .ant-input-affix-wrapper,
+  .ant-input-number,
+  .ant-radio-inner,
+  .ant-picker {
+    background-color: var(--background-l3);
+    background: var(--background-l3);
+    border-color: var(--stroke-disabled);
+  }
```

Comment:
**question** shouldn't these styles just apply to every input in the app? Do we need to scope it to `wizard-create`?

Reply:
I've been scoping any of the new design stuff to create specifically to avoid any weird inconsistencies outside of create

Reply:
aight good call. I think we should avoid this in general though as we'll end up with maintainability nightmare if we start classing everything too heavily.